PocketKit/Sources/PocketKit/Article/ReadableViewModel/ReadableViewModel.swift

```diff
@@ -108,4 +112,36 @@ extension ReadableViewModel {
         let event = SnowplowEngagement(type: .general, value: nil)
         tracker.track(event: event, contexts)
     }
+
+    func webViewActivityItemsForSavedItems(item: SavedItem) -> [UIActivity] {
+        let archiveActivityTitle: WebActivityTitles = (item.isArchived
+                                                       ? WebActivityTitles.moveToSaves
+                                                       : WebActivityTitles.archive)
```

Comment:
```suggestion
                                                       ? .moveToSaves
                                                       : .archive)
```

We shouldn't have to explicitly use the type, here, since the variable declaration declares the type (the compiler should infer the type).